apps/chat/channels.py

```diff
@@ -354,13 +356,17 @@ def new_user_message(self, message: BaseMessage) -> ChatMessage:
 
             try:
                 with self.trace_service.trace(
+                    experiment_id=self.experiment.id,
```

Comment:
What about moving this to the constructor and setting it when calling `TracingService.create_for_experiment`?